templates/patterns/navbar/navbar.ui_patterns.yml

```diff
@@ -2,63 +2,94 @@ navbar:
-      preview: false
-    placement:
+    expand:
       type: select
       label: Placement
       options:
-        fixed_top: fixed top
-        fixed_bottom: fixed bottom
-        sticky_top: sticky top
-    attributes:
-      type: attributes
-      label: Attributes
+        sm: small
+        md: medium
+        lg: large
+      preview: lg
+    color_set:
+      type: select
+      label: Available set of color
+      options:
+        light: light
+        dark: dark
+      preview: light
   fields:
     navbar_branding:
-      type: text
-      label: Brand title
-      description: Your company, product, or project name.
+      type: render
+      label: Navigation bar branding
+      preview:
+        - type: html_tag
+          tag: a
+          value: "Navbar"
+          attributes:
+            href: '#navbar'
+            rel: home
+            class: "navbar-brand"
+
+    navbar_right:
+      type: render
+      label: Navigation bar right
       preview:
         - type: html_tag
-          tag: span
-          value: "Brand"
+          tag: form
+          attributes:
+            class: "d-flex"
+          children:
+            '<label for="inlineFormInputGroupSearch" class="form-label visually-hidden">Search</label><input class="form-control" type="text" id="inlineFormInputGroupSearch" required="true" placeholder="Search"/><div class="ms-2"><button class="btn btn-primary btn-md" type="Search">Search</button></div>'
     navbar_left:
       type: render
-      label: Left content of navbar (main menu)
-      description: Full-height and lightweight navigation (including support for dropdowns).
+      label: Navigation bar left
       preview:
         - type: pattern
-          id: nav
-          settings:
-            attributes: "class='navbar-nav'"
+          id: link
+          label: "I'm a link"
+          path: "#first"
+          attributes:
+            class: "nav-link"
+        - type: pattern
+          id: link
+          label: "I'm a link 2"
+          path: "#second"
+          attributes:
+            class: "nav-link"
+        - type: pattern
+          id: link
+          label: "I'm a link 3"
+          path: "#third"
+          attributes:
+            class: "nav-link"
+        - type: pattern
+          id: dropdown
+          variant: default
           fields:
-            items: 
-              - title: Home
-                url: "#"
-                in_active_trail: true
-              - title: Library
-                url: "#"
-                is_expanded: true
-                below:
-                  - title: Sub 1
-                    url: "#"
-                  - title: Sub 2
-                    url: "#"
-              - title: Data
-                url: "#"
-    navbar_right:
-      type: text
-      label: Text
-      description: Vertically centered strings of text.
-      preview: "Navbar text with an inline element"
+            label: "Dropdown Toggle"
+            items:
```

Comment:
This has to be updated after dropdown is merged. This will be called dropdown-items. See https://github.com/openeuropa/oe_bootstrap_theme/pull/50